addons/l10n_pe/models/account_move.py

```diff
@@ -32,6 +43,17 @@ def _inverse_l10n_latam_document_number(self):
             number = rec.l10n_latam_document_number.split("-")
             rec.l10n_latam_document_number = "%s-%s" % (number[0], number[1].zfill(8))
 
+    @api.depends('l10n_latam_available_document_type_ids')
+    def _compute_l10n_latam_document_type(self):
+        result = super()._compute_l10n_latam_document_type()
+        for move in self.filtered(
+                lambda move: move.state == 'draft' and
```

Comment:
Sorry @jco-odoo 

I do not understand, all the conditions?

Reply:
Why not?  Is the default option we have now still an option or not?  Because we could immediately put it in the domain of all document types instead of just adapting the default one.  In other words, is there a need for the compute?